src/org/vivecraft/VSE.java

```diff
@@ -7,12 +7,7 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Can we like not mess with all the imports?

Reply:
yes, sorry, I suspect my IDE did that.

Reply:
Yeah if you're using idea, it defaults to using asterisk for more than I think 2 or 3 imports.

Reply:
Gotcha!